### ISSUE TITLE
switch to mysql

### ISSUE THREAD

**Issue by jamemackson** [hairbrain/gallery13#17]
- [x] update sails config to reference mysql
- [ ] establish dev vs prod connection details
- [x] setup mysql on server

**Comment by jamemackson** [comment]
I was reading somewhere that sails (waterline actually im guessing) doesn't have the ability to setup a local vs prod connection info so we might have to juggle that somehow until that is in place, or create a PR for it.  

but I have mysql installed on the server now.  what I did is [documented here](https://github.com/hairbrain/hairbrain.devops/blob/master/work-instructions/ubuntu-mysql-setup.md) 

@IanVS what do you recommend for updating that site with our new changes?  Should we just manually pull new versions down and rebuild the site or do you think we can script this out easily enough?  Maybe setup a simple rsync deploy using circleci or since we're not in as big of a hurry should we try to prototype out a [ember-cli-deploy](https://www.npmjs.com/package/ember-cli-deploy) deployment scheme?

**Comment by IanVS** [comment]
I think short term it will be easiest to ssh in and git pull new versions. I would like to get continuous deployment working first on hairbrain, since it is already set up with circle ci and CodeDeploy.  After that's up and running, it should be a simple matter to implement on gallery13 also should we find it to be necessary.

**Comment by jamemackson** [comment]
yea, that sounds good.  keep it simple jame, keep it simple ;)

**Comment by IanVS** [comment]
The biggest issue I see right now either ember-cli-deploy, is that it's necessary to have a deploy.json file containing AWS secrets. That makes me nervous, and I'm not sure how we could then integrate the work flow into circle ci. That's what I'm looking into now.

**Comment by jamemackson** [comment]
yes, that's definitely something to consider.  I want to say they mentioned something about that in the [EmberConf talk](https://www.youtube.com/watch?v=4EDetv_Rw5U&index=5&list=PLE7tQUdRKcyacwiUPs0CjPYt6tJub4xXU) about it.  But I don't remember if they had a solution already in place for that or if that was on the roadmap.  I'll have to go back and watch that again...

**Comment by IanVS** [comment]
Ah, now I see.  The secrets are populated by environment variables, which can be defined in the circle ci project settings.

**Comment by jamemackson** [comment]
awesome!

**Comment by jamemackson** [comment]
I got the site working against MySql on the gallery server.  I set the connection details (for now) in `./server/config/connections.js`  (I'll create a separate task for switching this to environment variables (or the .env file when present). ) 

I also had to import a copy of the database as sails doesn't create the database in production like it does locally.  I am using MySql Workbench connected via ssh to the server to administer the database.
